e2e/tests/dry-run/dry-run-test.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash -e
+
+# shellcheck disable=SC2086
+# shellcheck disable=SC2223
+
+TMPDIR="$(mktemp -d)"
+trap 'rm -rf -- "$TMPDIR"' EXIT
+
+# Run install --dry-run command and move output to tmpdir
+kubectl storageos install --dry-run
+cp -r storageos-dry-run $TMPDIR/ && rm -r storageos-dry-run
+# Fetch operator-manifests and store in temp dir
+docker run "storageos/operator-manifests:v2.5.0" > ${TMPDIR}/storageos-operator.yaml
```

Comment:
I think it's easy to get into a situation where the operator manifests differ here.

If we change the manifest in a later version, the plugin will install the latest version and then this will fail